src/helper-functions/auth-helpers.js

```diff
@@ -0,0 +1,27 @@
+import axios from "axios";
+
+
+const loginHandler = async (e, boolFunc, email, pass, navigate) => {
+    e.preventDefault();
+    try {
+        const response = await axios.post(`/api/auth/login`, {
+            email: "gkool@neog.camp",
```

Comment:
it should mention in  your pr description that no matter what, only a default user is going to get logged in , in my opinion.